src/Test/L0/Util/IOUtilL0.cs

```diff
@@ -673,6 +673,7 @@ public void DeleteFile_IgnoresDirectory()
         [Trait("Category", "Common")]
         [Trait("SkipOn", "darwin")]
         [Trait("SkipOn", "linux")]
+        [Trait("SkipOn", "freebsd")]
```

Comment:
Do these `SkipOn` things have a negation form?  All of these are trying to write 'not-windows' in increasingly verbose ways.  Can the be rewritten to make that intention clear?

Reply:
The test filters are created in [dev.sh](https://github.com/microsoft/azure-pipelines-agent/blob/fcca8a7f999d62b8e8646bffdbd60f6fe0f18ccd/src/dev.sh#L136) using the current platform, for example:
```
TestFilters="Level=L0&SkipOn!=${CURRENT_PLATFORM}"
```
We could change this to `TestFilters="Level=L0&OnlyOn=${CURRENT_PLATFORM}"`, but then we would probably have to add multiple `OnlyOn` attributes to all test methods, which would be even more confusing, I think.